### PR TITLE
Ajive revamp

### PR DESCRIPTION
**Pull request by rflperry**
#### What does this implement/fix? Explain your changes.
AJIVE was not working properly and didn't fit with our updated agreed upon API. The code also contained utility and organizational layout that needed improvement. Lots of items were moved to the same `decomposition.ajive` file and many things were deleted that were extraneous. Tutorial updated as well to reflect this corrected version.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/Users/rflperry/opt/miniconda3/envs/mvlearn-dev/bin/python"
+}
```
Comment (agramfort) [inline .vscode/settings.json:3]: to remove as the DSStore file.

I assume you do `git add .` it's quite dangerous

**Comment by agramfort** [comment]
@rflperry can you open PRs from your fork and not from the upstream repo? Otherwise it forces us to cleanup the merged branches and to avoid cluttering the upstream repo.

**Comment by rflperry** [comment]
Okay, will reopen PR on a fork.

**Comment by agramfort** [comment]
thanks

>

**Comment by agramfort** [comment]
don't forget to delete the branch
https://github.com/mvlearn/mvlearn/tree/ajive_revamp

>

**Pull request closed without merging**